### ISSUE TITLE
When outputting info about broken link, one should also consider outputting link text

### ISSUE THREAD

**Issue by hmol** [hmol/LinkCrawler#2]

**Comment by josedusol** [comment]
@hmol What you mean by "link text"?, the visible part of the link?.

I think in that case the return type for `List<string> GetValidUrlListFromMarkup(...)` would be changed to something like `List<Tuple<string,string>>` (or maybe some model class to avoid Tuple) because also the innerHtml of every `<a...>link text</a>` tag is needed for output. The same change would be needed for `GetAllUrlsFromMarkup(...)` and `GetAllUrlsFromHtmlDocument(...)`.

**Comment by hmol** [comment]
Yeah, this is an example of linktext `<a href="http://www.the-page.com/subpage>Here is linktext</a>"`.
I would avoid Tuple because of readability. Better to use a simple model class yes.

**Comment by josedusol** [comment]
Ok, i will look into it when i find time.

**Comment by nedruk** [comment]
Hi team, i'd like to have a look at this one if you don't mind. A question - what should be displayed if innerHtml is some other element, e.g. svg?

**Comment by Ravikc** [comment]
Why is this issue still open?
